docs/sources/setup-grafana/configure-grafana/enterprise-configuration/index.md

```diff
@@ -179,12 +179,16 @@ Keep dashboard content in the logs (request or response fields). This can signif
 
 ### verbose
 
-Log all GET requests and always include request body for generic POST/PUT/PATCH requests.
+Keep requests and responses body. This can significantly increase the size of your logs.
```

Comment:
thought: I think it might be worth explicitly calling out the fact that all requests (including `GET` requests) get logged when this is specified. What do you think about something like:
```suggestion
Log all requests and keep requests and responses body. This can significantly increase the size of your logs.
```